libs/cypress-indexeddb/src/lib/object-store-CRUD.ts

```diff
@@ -0,0 +1,150 @@
+import { createDatabaseConnection } from './open-database';
+import Log = Cypress.Log;
+
+type SetItemOperation = 'create' | 'update';
+type ReadDeleteOperation = 'read' | 'delete';
+type StoreOperation = keyof Pick<IDBObjectStore, 'get' | 'put' | 'delete'>;
+type ConsolePropObject = {
+  key: string;
+  value?: unknown;
+  error?: Error;
+};
+
+export function readItem<T>(store: IDBObjectStore, key: string): Promise<T> {
+  const { log, consoleProps } = createCRUDLog('read', key);
+  if (store?.constructor?.name !== 'IDBObjectStore') {
```

Comment:
maybe we could reuse the function from above. Just extract it to a separate file

```suggestion
  if (!isIDBObjectStore(store)) {
```